src/plugins/analysis/cve_lookup/test/test_setup_repository.py

```diff
@@ -1107,30 +1107,28 @@ def test_exists(monkeypatch):
 
 def test_extract_relevant_feeds():
     sr.DATABASE = sr.DatabaseInterface(PATH_TO_TEST + 'test_update.db')
-    assert [('CVE-2018-0002', 2018), ('CVE-2018-0003', 2018)] == sorted(
-        sr.extract_relevant_feeds(from_table='new', where_table='outdated')
-    )
+    assert sorted(sr.extract_relevant_feeds(from_table='new', where_table='outdated')) == [
+        ('CVE-2018-0002', 2018),
+        ('CVE-2018-0003', 2018),
+    ]
 
 
 def test_delete_outdated_feeds():
+    sr.DATABASE = sr.DatabaseInterface(PATH_TO_TEST + 'test_update.db')
```

Comment:
Shouldnt monkeypatch be used for this kind of stuff?

Reply:
Probably. The whole module shouldn't be structured like it is but I saw it as out of scope to fix it in this PR

Reply:
As discussed lets leave it as it is.